samples/ORKCatalog/ORKCatalog/Tasks/TaskListRow.swift

```diff
@@ -158,6 +160,10 @@ enum TaskListRow: Int, CustomStringConvertible {
                     .TwoFingerTappingInterval,
                     .WalkBackAndForth,
                 ]),
+            TaskListRowSection(title: "Video Instruction", rows:
```

Comment:
I think the section should be called Miscellaneous, so that it can serve as a bucket for future misc steps as well.